### ISSUE TITLE
NullPointerExceptions in UpdateFlowSpace

### ISSUE THREAD

**Issue by nikr** [opennetworkinglab/flowvisor#249]
When UpdateFlowSpace is called without a value for the priority a NullPointerException is thrown. 

the problem can be reproduced by sending e.g. the JSON message 

```
{
"method":"update-flowspace",
"id":"update-flowspace887",
"params":[
  { 
  "name":"flowspace_foo",
  "dpid":"00:00:00:00:00:00:00:05",
  "match":{"in_port":2}
  }
],
"jsonrpc":"2.0"
}
```

to FlowVisor's JSON RPC. 

A patch to remedy this is [here](https://gist.github.com/nikr/5874069)

**Comment by alshabib** [comment]
Thanks for the patch and well spotted issue. It'll be applied soon and we'll let you know. 

## 

Ali Al-Shabibi (sent from handheld)

**Comment by nikr** [comment]
fixed in 30fbc06
